src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDocletWriter.java

```diff
@@ -1678,68 +1678,23 @@ public boolean isCoreClass(TypeElement typeElement) {
     }
 
     /**
-     * Adds the annotation types for the given packageElement.
+     * Return a content tree containing the  annotation types for the given Element.
```

Comment:
(minor) either put type names (Element) in code font or use English equivalent (element).